R/scale-.R

```diff
@@ -1398,8 +1385,11 @@ check_transformation <- function(x, transformed, name, arg = NULL, call = NULL)
   cli::cli_warn(msg, call = call)
 }
 
-trans_support_nbreaks <- function(trans) {
-  "n" %in% names(formals(trans$breaks))
+support_nbreaks <- function(fun) {
+  if (inherits(fun, "ggproto_method")) {
+    fun <- environment(fun)$f
+  }
```

Comment:
Might need to deal with ggproto methods as `self$breaks` can be a function that gets wrapped by `ggproto()`